test/data/executor.start.yaml

```diff
@@ -1,3 +1,5 @@
+annotations:
+    beta.screwdriver.cd/executor: k8s
```

Comment:
I think we'll have a list of recognized annotations much like Kubernetes does:
https://kubernetes.io/docs/reference/labels-annotations-taints/